src/app/core/services/http/http.service.ts

```diff
@@ -85,7 +85,7 @@ export class HttpService {
         let result: any = data.data;
         if(result?.meta?.data?.length && !this.isFeedbackTriggered){
           this.isFeedbackTriggered = true;
-          // this.openModal(result?.meta?.data[0]);
+          this.openModal(result?.meta?.data[0]);
```

Comment:
Check if there is data. length

Reply:
data.length already checking in if condition. do we need to check again. ?